compute/oslogin/oslogin_service_account_ssh_test.py

```diff
@@ -116,16 +125,18 @@ def ssh_firewall():
     firewall_client = compute_v1.FirewallsClient()
     firewall_client.insert(request).result()
 
-    yield firewall_client.get(project=PROJECT, firewall=TEST_ID)
     try:
-        firewall_client.delete(project=PROJECT, firewall=TEST_ID)
-    except (NotFound, BadRequest):
-        # That means the GCE Enforcer deleted it before us
-        pass
+        yield firewall_client.get(project=PROJECT, firewall=TEST_ID)
+    finally:
+        try:
+            firewall_client.delete(project=PROJECT, firewall=TEST_ID)
+        except (NotFound, BadRequest):
+            # That means the GCE Enforcer deleted it before us
+            pass
```

Comment:
```suggestion
            print("GCE Enforcer already deleted the rule") 
```

I know this was here before, but given that we are fixing around this, it's always good to remove a pass in an except clause